src/main/cpp/t06_sum.cpp

```diff
@@ -22,7 +22,20 @@
 
 using namespace std;
 
-int t06_sum() {
+int fun(int t, int sum){
+    if(t == 0)
+        return sum;
+    else{
+        sum += t;
+        cin >> t;
+        sum = fun(t, sum);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'sum' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4513105)